device-registry/src/main/scala/org/genivi/sota/device_registry/Routes.scala

```diff
@@ -105,7 +118,9 @@ class Routes(namespaceExtractor: Directive1[Namespace],
   def api: Route =
     handleExceptions(ExceptionHandler(Errors.onMissingDevice orElse Errors.onConflictingDevice
```

Comment:
Also have a solution for this mess in one of my branches but this is enough for now.